### PR TITLE
Awesome books using ES6 Pull Request

### PR DESCRIPTION
**Pull request by henask12**
- [ ] This pull request re-creates the Awesome books app using ES6 syntax, modules, and the Luxon library. The app now has the following features:

- Uses ES6 syntax.

- Divided into modules.

- Imports modules into the index.js file.

- Displays the current date below the navigation bar.

- [ ] `Changes`

> The following changes have been made to the code:

- The code has been converted to use ES6 syntax. This includes using arrow functions, let and const, and other ES6 features.

- The code has been divided into modules. This makes the code easier to read, understand, and maintain.

- The modules have been imported into the index.js file. This allows the code to be modularized and reused.

- The current date is displayed below the navigation bar using the Luxon library. This library provides a simple way to get the current date and time.

### CODE REVIEW

File: modules/addBooks.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Book from './books.js';
+
+export default function addBooks(event) {
```
Comment (Lucas-Masaba) [inline modules/addBooks.js]: - [ ] Please use the `arrow syntax` here. We are using ES6 Javascript in this milestone and every function should be an arrow function.

File: modules/displayBooks.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export default function displayBooks() {
```
Comment (Lucas-Masaba) [inline modules/displayBooks.js]: - [ ] Please use the `arrow syntax` here. We are using ES6 Javascript in this milestone and every function should be an arrow function.

File: modules/showSection.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export default function showSection(sectionId) {
```
Comment (Lucas-Masaba) [inline modules/showSection.js]: - [ ] Please use the `arrow syntax` here. We are using ES6 Javascript in this milestone and every function should be an arrow function.

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @henask12  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks are passing :heavy_check_mark: 
- Desriptive pull request message :writing_hand: 
- The app is functioning as expected :gear: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @henask12 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ ES6 syntaxes are correctly used

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**